python/private/patch_whl.bzl

```diff
@@ -73,11 +73,15 @@ def patch_whl(rctx, *, python_interpreter, whl_path, patches, **kwargs):
         parsed_whl.platform_tag,
     ]))
 
+    record_patch = rctx.path("RECORD.patch")
```

Comment:
Does this work if RECORD.patch doesn't exist? In chat the other day, Jeremy linked to an issue where it sounded like the initial path() call would fail. Or maybe that's only if a label is passed? relative paths are OK?

Reply:
Resolved in chat